openslides_backend/shared/env.py

```diff
@@ -1,28 +1,99 @@
-import os
+from enum import Enum, auto
+from typing import Any, Dict
 
-from .exceptions import ServerError
-
-INTERNAL_AUTH_PASSWORD_FILE = "INTERNAL_AUTH_PASSWORD_FILE"
-OPENSLIDES_DEVELOPMENT = "OPENSLIDES_DEVELOPMENT"
 DEV_PASSWORD = "openslides"
 
 
+class Loglevel(Enum):
+    NOTSET = auto()
+    DEBUG = auto()
+    INFO = auto()
+    WARNING = auto()
+    ERROR = auto()
+    CRITICAL = auto()
+
+
 def is_truthy(value: str) -> bool:
     truthy = ("1", "on", "true")
     return value.lower() in truthy
 
 
-def is_dev_mode() -> bool:
-    dev = os.environ.get(OPENSLIDES_DEVELOPMENT, "off")
-    return is_truthy(dev)
+class Environment:
+    """
+    Contains all environment variables. See the vars attribute for all defaults.
+    """
+
+    vars = {
+        "ACTION_PORT": "9002",
```

Comment:
I miss the email settings

Reply:
I will have a look

Reply:
These are only the defaults; email settings previously did not have default settings, either. So this should be fine.